Applications/ClientWebPortal/Source/webapp/src/utils/value-validator/index.js

```diff
@@ -0,0 +1,8 @@
+class ValueValidator {
```

Comment:
I don't see why this is a singleton class.  Does JavaScript not support plain functions?

Reply:
It doesn't need to be singleton. I use it mostly for utility classes that are stateless and and no point in allowing anyone to instantiate in multiple places.